src/reference/asciidoc/ip.adoc

```diff
@@ -99,6 +99,26 @@ For even more reliable networking, TCP can be used.
 
 Starting with _version 4.3_, the `ackPort` can be set to `0`, in which case the Operating System chooses the port.
 
+Another way of configuring outbound channel adapter is to make it use other UDP channel adapters socket.
+Outbound channel adapter can use e.g. inbound channel adapters socket to send datagrams through same port which they were received. It's useful in a scenario when spring application works as UDP server and clients operate behind NAT.
+To configure outbound channel adapter with external socket the `socket-expression` parameter must be used with proper SpEL expression.
```

Comment:
The `socket-expression` must be mentioned in the `whats-new.adoc`.
Right, your doc requires some polishing, but it isn't critical: @garyrussell polishes my docs, too :wink: